dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1249,9 +1265,72 @@ public void loadDockerImages() {
 
     public abstract Client getClient();
 
+
+
     /**
      * help text output
      */
+    public void wesHelp() {
+        printHelpHeader();
+        out("Commands:");
+        out("");
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " wes --help");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes launch [parameters]");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes status [parameters]");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes cancel [parameters]");
+        out("");
+        out("Description:");
+        out(" Executes a command on a Workflow Execution Service (WES) endpoint.");
+        printWesHelpFooter();
+        printHelpFooter();
+    }
+
+    protected void wesLaunchHelp() {
+        printHelpHeader();
+        out("Usage: dockstore wes " + getEntryType().toLowerCase() + " launch --help");
+        out("       dockstore wes " + getEntryType().toLowerCase() + " launch [parameters]");
+        printLaunchHelpBody();
+        printWesHelpFooter();
+        printHelpFooter();
+    }
+
+    public void wesStatusHelp() {
+        printHelpHeader();
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " wes status --help");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes status [parameters]");
+        out("");
+        out("Description:");
+        out("  Status, gets the status of a " + getEntryType() + ".");
+        out("Required Parameters:");
+        out("  --id <id>                           Id of a " + getEntryType() + " at the WES endpoint, e.g. id returned from the launch command");
+        out("Optional Parameters:");
+        out("  --verbose                           Provide extra status information");
+        out("");
+        printWesHelpFooter();
+        printHelpFooter();
+    }
+
+    public void wesCancelHelp() {
+        printHelpHeader();
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " wes cancel --help");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes cancel [parameters]");
+        out("");
+        out("Description:");
+        out("  Cancels a " + getEntryType() + ".");
+        out("Required Parameters:");
+        out("  --id <id>                           Id of a " + getEntryType() + " at the WES endpoint, e.g. id returned from the launch command");
+        out("");
+        printWesHelpFooter();
+        printHelpFooter();
+    }
+
+    public void printWesHelpFooter() {
+        out("Global Optional Parameters:");
+        out("  --wes-url <WES URI>                 WES URI where the WES command should run, e.g. 'http://localhost:8080/ga4gh/wes/v1'");
```

Comment:
I don't like the word "command" in  "where the WES command should run". If anything, you're sort of running the (client) command on your local box. How about `The URL of the WES server, e.g., ....`.

Also, I think you should say URL, just to be consistent with `--wes-url`; right now it's a mixture of url and URI.

Reply:
changed URI to URL